python/util/templates.py

```diff
@@ -66,6 +66,13 @@ def generate_react_components(self):
         script = """
 document.addEventListener("DOMContentLoaded", function () {
     'use strict';
+    var components,
+        baseUserEmbed = {'profilePicture': {},
+                         'emailAddresses': {} },
+        baseJobEmbed = {},
```

Comment:
@DanielCollins we're going to need to work out a solution where we can set this baseJobEmbed to the original embed used to fetch the job by the python view.